python/ray/util/client/api.py

```diff
@@ -277,6 +277,14 @@ def get_runtime_context(self):
         """
         return ClientWorkerPropertyAPI(self.worker).build_runtime_context()
 
+    def get_current_runtime_env(self):
+        """Get the runtime env of the current client/driver.
```

Comment:
How does this differ from the `get_current_runtime_env` in `runtime_env.py`?  Does the other one return the runtime env of the current task/actor/job, while this one only returns the runtime env installed by the Ray client server (the job-level runtime env)?

Reply:
Yes, I think you are right!